lib/widgets/transaction_item.dart

```diff
@@ -32,15 +32,17 @@ class TransactionItem extends StatelessWidget {
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
           Text(
-            transaction.goalName ?? 'Christ Pres Chruch -f',
+            transaction.destinationName,
             maxLines: 1,
             overflow: TextOverflow.ellipsis,
             style: TextStyle(
               color: Color(0xFF3B3240),
               fontWeight: FontWeight.bold,
             ),
           ),
-          SizedBox(height: 5,),
+          SizedBox(
```

Comment:
Something that I've noticed in the whole codebase is that the "const" is not used for non-changing widgets, like this. If you add "const" it will make your code faster. When rebuilding the tree the compiler knows that this widget doesn't have to be rebuilt

Reply:
good point, thanks. Going to include this change to the cleanup task [DK-493]

Reply:
You can do this with the constructors as well, it will have the same efffect